engine/apps/telegram/tasks.py

```diff
@@ -215,3 +217,19 @@ def on_create_alert_telegram_representative_async(self, alert_pk):
     )
     for message in messages_to_edit:
         edit_message.delay(message_pk=message.pk)
+
+
+@shared_dedicated_queue_retry_task(
+    autoretry_for=(Exception,), retry_backoff=True, max_retries=1 if settings.DEBUG else None
+)
+def on_alert_group_action_triggered_async(log_record_id):
```

Comment:
Just in case, don't forget to add the task to the celery routes setting.

Reply:
thank you, we a really cool test `common/tests/test_task_queue_assignment.py`